pkg/config/validation/validation.go

```diff
@@ -1121,21 +1121,20 @@ var ValidateSidecar = registerValidateFunc("ValidateSidecar",
 				if strings.HasPrefix(i.DefaultEndpoint, UnixAddressPrefix) {
 					errs = appendValidation(errs, ValidateUnixAddress(strings.TrimPrefix(i.DefaultEndpoint, UnixAddressPrefix)))
 				} else {
-					// format should be 127.0.0.1:port or :port
-					parts := strings.Split(i.DefaultEndpoint, ":")
-					if len(parts) < 2 {
```

Comment:
why remove it

Reply:
should not use `strings.Split` the `:` to split the host ip and port, because `:`  is included in IPv6 address